diesel_cli/src/print_schema.rs

```diff
@@ -3,15 +3,20 @@ use crate::config;
 use crate::infer_schema_internals::*;
 use serde::de::{self, MapAccess, Visitor};
 use serde::{Deserialize, Deserializer};
+use serde_regex::Serde as RegexWrapper;
 use std::error::Error;
 use std::fmt::{self, Display, Formatter, Write};
 use std::io::Write as IoWrite;
 
 const SCHEMA_HEADER: &str = "// @generated automatically by Diesel CLI.\n";
 
+type Regex = RegexWrapper<::regex::Regex>;
+
 pub enum Filtering {
```

Comment:
I'm not sure why this enum now has 5 variants instead of just transforming the old ones into variants that use an regex internally?

Reply:
Can I use `Filtering::OnlyTableRegexes` for `whitelist` and `Filtering::ExceptTableRegexes` for `blacklist`? These flags have been deprecated. If I can, I will remove `Filtering::OnlyTables` and `Filtering::ExceptTables`.

Reply:
As the next release will be diesel 2.0 I think we can remove the support for `whitelist`/`blacklist` altogether in favor of the `only-table`/`expect-table` support.

Reply:
Ok. So,  I'm going to transform the old ones into variants that use an regex!

Reply:
I've just done it!